lib/xcode.js

```diff
@@ -136,20 +128,20 @@ const getVersionMemoized = _.memoize(
 );
 
 async function getVersion (parse = false, retries = DEFAULT_NUMBER_OF_RETRIES, timeout = XCRUN_TIMEOUT) {
-  let version = await getVersionMemoized(retries, timeout);
+  const version = await getVersionMemoized(retries, timeout);
+  // xcode version strings are not exactly semver string: patch versions of 0
+  // are removed (e.g., '10.0.0' => '10.0')
+  const versionString = version.patch > 0 ? version.version : `${version.major}.${version.minor}`;
```

Comment:
what is returned as `patch` if the last number is missing?

Reply:
```
 $ node
> const semver = require('semver')
undefined
> semver.coerce('10.0')
SemVer {
  loose: undefined,
  raw: '10.0.0',
  major: 10,
  minor: 0,
  patch: 0,
  prerelease: [],
  build: [],
  version: '10.0.0' }
```